server/src-lib/Hasura/RQL/Types/SchemaCacheTypes.hs

```diff
@@ -11,7 +11,7 @@ import           Data.Aeson.Types
 import           Data.Text.NonEmpty
 
 import           Hasura.Backends.Postgres.SQL.Types
-import           Hasura.RQL.Types.Common
+import           Hasura.RQL.Types.Common             hiding (ConstraintName)
```

Comment:
I guess this is needed because Postgres's `ConstraintName` is used in this module? It looks like this is because `TableObjId` etc uses `ConstraintName`. I guess we'll have to move these types too to the Postgres's specific module. (Just a note)

Reply:
My bad, already mentioned in an earlier comment.

Reply:
Correct, yeah. There's a lot to clean in `RQL.Types`, and we will do that in due time. :)